pkg/services/ngalert/api/api_convert_prometheus.go

```diff
@@ -494,3 +497,10 @@ func namespaceErrorResponse(err error) response.Response {
 
 	return toNamespaceErrorResponse(err)
 }
+
+func getProvenance(ctx *contextmodel.ReqContext) models.Provenance {
+	if _, disabled := ctx.Req.Header[disableProvenanceHeaderName]; disabled {
+		return models.ProvenanceNone
+	}
+	return models.ProvenanceConvertedPrometheus
+}
```

Comment:
Same as in the provisioning API but different header:

https://github.com/grafana/grafana/blob/e933f7cf011ce045ab999fa6cb7e40eb8dc7d962/pkg/services/ngalert/api/api_provisioning.go#L540-L545